src/openrct2/world/Map.cpp

```diff
@@ -2396,6 +2396,23 @@ TileElement* map_get_track_element_at_with_direction_from_ride(const CoordsXYZD&
     return nullptr;
 };
 
+WallElement* map_get_wall_element_at(const CoordsXYRangedZ& coords)
+{
+    auto tileElement = map_get_first_element_at(coords);
+
+    if (tileElement != nullptr)
+    {
+        do
+        {
+            if (tileElement->GetType() == TILE_ELEMENT_TYPE_WALL && coords.baseZ < tileElement->GetClearanceZ()
+                && coords.clearanceZ > tileElement->GetBaseZ())
+                return tileElement->AsWall();
```

Comment:
```suggestion
            {
                return tileElement->AsWall();
            }
```

Let's add the braces because it looks clearer

Reply:
Added!